harvard-lts/CURIOSity
# Push to GitHub 9 15

**Push of gitlab main to GitHub for 9-15*
* * *

# What does this Pull Request do?
Includes some security patches, and work for the ticket to show result counts on search pages (ticket [#179](https://github.com/harvard-lts/CURIOSity/issues/179)). Also a new version fo the oai gem in Gemfile.lock

# How should this be tested?

Security patches work if the site comes up/functions.

To test page count, perform a search -- you should see at the top of the search results now something like 'Showing 5 of 25 results'

@phil-plencner-hl I believe you worked on the oai stuff, I am not sure what change that put through.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
